pkg/printutil/printutil.go

```diff
@@ -35,6 +35,18 @@ type Table struct {
 	// Function which will eval whether to apply color to a col
 	// ColorColCond    func(t *Table) (bool, error)
 	// ColorColCode [2]string
+
+	altPadding []int
+
+	headerPadding []int
+
+	DynamicPadding bool
+}
+
+// comment
+type TempRow struct {
```

Comment:
It may make sense to remove this TempRow struct and instead combine the attributes you need into the Row struct. This will allow you to avoid passing around multiple row types and instead modify rows in place.